app/assets/javascripts/backbone/views/tab_history_manager.coffee

```diff
@@ -0,0 +1,40 @@
+# Handles tabs interaction with browser history
+class MS.Views.TabHistoryManager extends Backbone.View
+
+  initialize: (params) ->
+    @listenTo(Backbone, 'popstate', @popstate)
+    @basePath = params.basePath
+    @firstTab = @$('a[role=tab]').first().data('tab-id')
+    activeTab = @$('li.tab.active a[role=tab]').data('tab-id')
+    @showTab(activeTab)
+
+  events:
+    'click a[role=tab]': 'tabClicked'
+
+  popstate: (e) ->
+    @$("a[role=tab]").blur()
```

Comment:
Otherwise the old tab stays focused and has a gray bg. Can add a comment if you like.